src/api/lib/tasks/dev.rake

```diff
@@ -186,39 +186,53 @@ namespace :dev do
       require 'factory_bot'
       include FactoryBot::Syntax::Methods
 
+      # Users
       admin = User.where(login: 'Admin').first
```

Comment:
What happens if the user Admin doesn't exist? I will suggest something like you do in [line 192](https://github.com/openSUSE/open-build-service/pull/9413/files#diff-d1bd1d1c8aec8f6a9617aa0dbf3a8e01R192).

Reply:
I assume it's a strange case, but it can happen. Updating...